pyproject.toml

```diff
@@ -7,14 +7,15 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = ">=3.8.1,<4.0"
+python = ">=3.8.1,<3.12"
 httpx = ">=0.15.4,<0.26.0"
 pydantic = ">=1.10.7, <3.0"
 backoff = "^2.2.1"
 openai = ">=0.27.8"
 wrapt = "1.14"
 langchain = { version = ">=0.0.309", optional = true }
 chevron = "^0.14.0"
+llama-index = {version = "^0.10.6", optional = true}
```

Comment:
Do you know by any chance ntil when this implementation is stable? It would be great to also support versions which are a couple of months old.